src/style.css

```diff
@@ -0,0 +1,78 @@
+[data-qa="big"] {
+  font-family: Avenir, Book, sans-serif;
+  margin-top: 12px;
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 24px ;
+  line-height: 22px;
+  background-size: 19px;
+  padding-left: 62px;
+  font-weight: 400;
+  font-size: 16px;
+}
+
+[data-qa="big"]:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  color: #3d4e61;
+}
+
+[data-qa="big"]:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #ccc;
+  border-radius: 3px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 24px;
+  line-height: 22px;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+[data-qa="big"]::placeholder:default {
+  font-family: Avenir, Book, sans-serif;
+  margin: 27px 62px;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+}
+
+[data-qa="big"]::placeholder:focus {
+  margin: 27px 62px;
+  font-size: 16px;
+  line-height: 22px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+[data-qa="small"] {
+  font-family: Avenir, Book, sans-serif;
+  box-sizing: border-box;
+  width: 100%;
+  height: 42px;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 12px 14px;
+  background-size: 11px;
+  font-weight: 400;
+  padding-left: 33px;
+  padding-top: 3px;
+}
+
+[data-qa="small"]::placeholder {
+  font-family: Avenir, Book, sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
```

Comment:
check styles and remove repeatable, and which not necessary to set again ( small input, hover, focus states)
for example make class search, and search-small for small form, where reset only classes, which different from big form